lib/haml_lint/tree/tag_node.rb

```diff
@@ -198,5 +206,17 @@ def script
     def text
       (@value[:value] if @value[:parse]) || ''
     end
+
+    private
+
+    def parsed_attributes
+      HamlLint::RubyParser.new.parse(hash_attributes_source)
```

Comment:
also, the `parse_ruby` method is an instance method on `Linter`. Should I create a corresponding class method? It seems like adding code to the `Linter` class for the sake of `TagNode` is a smell. Should they be interacting in that way?